modules/hostap/src/supp_events.c

```diff
@@ -285,12 +285,20 @@ static enum wifi_link_mode get_sta_link_mode(struct wpa_supplicant *wpa_s, struc
 #ifdef CONFIG_WIFI_NM_HOSTAPD_AP
 static bool is_twt_capable(struct hostapd_iface *iface, struct sta_info *sta)
 {
+#ifdef CONFIG_IEEE80211AX
```

Comment:
Could we use Zephyr specific config symbol here?
```
#if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT_11AX)
...
```
It would feel more natural as this is a zephyr specific file.

Reply:
Yes, that is more appropriate.